xion/assetlist.json

```diff
@@ -0,0 +1,35 @@
+{
+  "$schema": "../assetlist.schema.json",
+  "chain_name": "xion",
+  "assets": [
+    {
+      "description": "The native staking token of the Xion network.",
+      "denom_units": [
+        {
+          "denom": "uxion",
+          "exponent": 0,
+          "aliases": ["microxion"]
+        },
+        {
+          "denom": "XION",
+          "exponent": 6,
+          "aliases": ["xion"]
+        }
+      ],
+      "base": "uxion",
+      "display": "XION",
+      "name": "xion",
+      "symbol": "XION",
+      "coingecko_id": "xion",
```

Comment:
This is not correct, should be xion-2
Use info in https://github.com/cosmos/chain-registry/pull/5657